frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -401,8 +421,45 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
     : groups,
   new Set<SpecCapability.arrayFieldGroup>());
 
+  const advancedFields = fields.filter(f => !f.capabilities.some(c => c.startsWith(SpecCapability.fieldGroup) || c.startsWith(SpecCapability.arrayFieldGroup)))
+    .filter(f => f.capabilities.includes(SpecCapability.advanced));
+
   useScrollToTopOnMount();
 
+  type FieldGroupProps = {
+    defaultExpand: boolean;
+    group: SpecCapability.fieldGroup;
+  };
+
+  const FieldGroup: React.FC<FieldGroupProps> = ({group, defaultExpand}) => {
+    const [expand, setExpand] = React.useState<boolean>(defaultExpand);
+
+    return <div key={group} id={group}>
+      <div className="co-operand-field-group-title">
+        <label className="form-label">{_.startCase(group.split(SpecCapability.fieldGroup)[1])}</label>
+        <button className="btn btn-link" onClick={e => {
```

Comment:
`type="button"`, otherwise the enter key triggers this button and won't submit the form. I think you can also remove `preventDefault` if `type="button"`.

Reply:
Ah that's why it was submitting the form when I clicked "Expand"!

Reply:
Done